app/models/likes.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_likes_counter
```

Comment:
- [ ] Same thing applies to this as to after_save.

Reply:
done